styles/nav/_mixins.scss

```diff
@@ -0,0 +1,41 @@
+@mixin link-tab() {
```

Comment:
i'm not sure. the name feels awkward and i'm not sure why. maybe `nav-link` would fit better? I think it's because `link-tab` is almost repetitive where with `nav-link` it would be specifically a link used for nav.